v2/spanner-to-sourcedb/terraform/samples/spanner-to-sharded-mysql/terraform.tfvars

```diff
@@ -0,0 +1,138 @@
+  # The project where the resources will be deployed
+  project = "<YOUR_PROJECT_ID>" # Replace with your GCP project ID
+  # The host project in case of Shared VPC setup
+  host_project = "<YOUR_HOST_PROJECT_ID>" # If you are using a shared VPC
+  # The region where the resources will be deployed
+  region = "us-central1"
+  # Optional ID for the migration process
+  migration_id = "migration-123"
+  # Optional name for the replication bucket (defaults to "rr-bucket")
+  replication_bucket = "my-replication-bucket"
+  # Optional flag to control adding policies to the service account (defaults to true)
+  add_policies_to_service_account = false
+}
+
+dataflow_params = {
+  template_params = {
+    # Optional name for the change stream
+    change_stream_name = "my-change-stream"
+    # The ID of the Spanner instance
+    instance_id = "<YOUR_SPANNER_INSTANCE_ID>" # Replace with your Spanner instance ID
+    # The ID of the Spanner database
+    database_id = "<YOUR_SPANNER_DATABASE_ID>" # Replace with your Spanner database ID
+    # Optional ID of the Spanner project
+    spanner_project_id = "<YOUR_SPANNER_PROJECT_ID>" # Replace with your Spanner project ID (if different from the main project)
+    # Optional ID of the metadata instance
+    metadata_instance_id = "<YOUR_METADATA_INSTANCE_ID>" # Replace with your metadata instance ID (if applicable)
+    # Optional ID of the metadata database
+    metadata_database_id = "<YOUR_METADATA_DATABASE_ID>" # Replace with your metadata database ID (if applicable)
+    # Optional start timestamp for replication
+    start_timestamp = "2024-10-01T00:00:00Z"
+    # Optional end timestamp for replication
+    end_timestamp = "2024-10-31T23:59:59Z"
+    # Optional prefix for shadow tables
+    shadow_table_prefix = "shadow_"
+    # Optional path to a local session file
+    local_session_file_path = "/path/to/session/file"
+    # Optional filtration mode
+    filtration_mode = "column-list"
+    # Optional path to a custom sharding JAR file
+    sharding_custom_jar_path = "/path/to/sharding/jar"
+    # Optional name of the custom sharding class
+    sharding_custom_class_name = "com.example.ShardingClass"
+    # Optional parameters for custom sharding
+    sharding_custom_parameters = "param1=value1,param2=value2"
+    # Optional timezone offset for the source database
+    source_db_timezone_offset = "+08:00"
+    # Optional DLQ GCS Pub/Sub subscription
+    dlq_gcs_pub_sub_subscription = "projects/<YOUR_PROJECT_ID>/subscriptions/my-subscription" # Replace with your project ID and subscription name
+    # Optional name of the directory to skip
+    skip_directory_name = "skip-directory"
+    # Optional maximum number of shard connections
+    max_shard_connections = "10"
+    # Optional dead letter queue directory
+    dead_letter_queue_directory = "gs://my-bucket/dlq"
+    # Optional maximum retry count for DLQ
+    dlq_max_retry_count = "5"
+    # Optional run mode
+    run_mode = "parallel"
```

Comment:
The only supported run modes are regular or retryDLQ afaik

Reply:
Fixed